app/views/layouts/_center_div_no_listnav.html.haml

```diff
@@ -10,9 +10,14 @@
             = miq_toolbar toolbar_from_hash
       .row#main-content.miq-layout-center_div_no_listnav{:class => @lastaction == "show_dashboard" || @layout == "monitor_alerts_overview" ? 'miq-body' : ''}
         .col-md-12
-        - if layout_uses_tabs?
-          .col-md-12
-            = render :partial => 'layouts/tabs'
+          .row
+            .col-md-7
+              - if layout_uses_tabs?
+                = render :partial => 'layouts/tabs'
+            .col-md-5
+              - if show_search?
```

Comment:
Here you have a condition, while `_center_div_with_listnav` doesn't. You're introducing a different dataflow for your introduced case and so increasing complexity.

Reply:
Ah, yes, this came to my mind, too, but on the other side, it does not make sense to me to put the same stuff also to `_center_div_with_listnav` just for being the "same" as `_center_div_no_listnav`. It would unnecessary increase the complexity regarding `_center_div_with_listnav`. Yes, it would be more consistent regarding the code in `center_div.. ` hamls, but I don't see any other pros.

Also for me it does not make sense to have such logic in `_center_div_with_listnav`, as we always display Search bar for screens with listnav. So I think that the 'feature' I am introducing in this PR, would remain unused in `_center_div_with_listnav`, we would have only one extra condition there, which would always have to be executed - and always with the same result.

Reply:
And what would you say about moving `if show_search?` to the `searchbar.html.haml`?